ignite/services/plugin/app_yml.go

```diff
@@ -0,0 +1,18 @@
+package plugin
+
+import (
+	semver "github.com/blang/semver/v4"
+)
+
+// AppYML is the structure of app.ignite.yml file.
+type AppYML struct {
+	Version semver.Version     `yaml:"version"`
```

Comment:
If `Version` represent app versions it might be better to be defined in `AppInfo` because the config might contain different apps which might have different versions.

Otherwise if `Version` is the config file version it might be simpler to use a number like [Ignite config](https://github.com/ignite/cli/blob/ffcfa6d9dc5133d9cb738b350533269c22c4afde/ignite/config/chain/base/config.go#L156).

Reply:
+1, AppYML, could as well get confused with this [app.yaml](https://docs.cosmos.network/main/build/building-apps/app-go-v2#alternative-formats)

Reply:
> If `Version` represent app versions it might be better to be defined in `AppInfo` because the config might contain different apps which might have different versions.
> 
> Otherwise if `Version` is the config file version it might be simpler to use a number like [Ignite config](https://github.com/ignite/cli/blob/ffcfa6d9dc5133d9cb738b350533269c22c4afde/ignite/config/chain/base/config.go#L156).

@Ehsan-saradar which version does the `Version` field represent? App version or config version?

Reply:
Oh, sorry, I just saw that you changed the version to `uint` so it's the config version